dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/commands/ExportBaseCommand.java

```diff
@@ -94,8 +94,8 @@ public abstract class ExportBaseCommand extends CamelCommand {
     protected String repos;
 
     @CommandLine.Option(names = {
-            "--dep", "--deps" }, description = "Add additional dependencies (Use commas to separate multiple dependencies).")
-    protected String dependencies;
+            "--dep", "--deps" }, arity = "*", description = "Add additional dependencies")
+    private String[] _dependencies; // [TODO] make less protected when we ditch --deps
```

Comment:
What prevents us from ditching `--deps` right away to avoid these TODOs?

Reply:
As I said ...
> The code would be a lot simpler when we got rid of --deps
> Not sure whether we can do this any time soon.

If the answer is "nothing", we could do this straight away, which I would prefer too (if possible).